### PR TITLE
add jquery-spellcheck@0.2 w/ git auto-update

### LINKED ISSUE #9892: [Request] Add jquery-spellcheck
**Library name:** jquery-spellcheck
**Git repository url:** https://github.com/brandonaaron/jquery-spellcheck
**License(s):** MIT

### PR DESCRIPTION
**Pull request by honestree**
close #9892, cc @brandonaaron

Pull request for issue: #9892
Related issue(s): # #
@kennynaoh please help me review this PR! Thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/brandonaaron/jquery-spellcheck
 * Official website (optional, not the repository): 
 * NPM package url (optional):
 * License and its reference: https://github.com/brandonaaron/jquery-spellcheck/blob/master/LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 94
   - Count of watchers: 6
   - Count of forks: 22
* NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-spellcheck/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "0.2",
+  "author": {
+    "name": "Brandon Aaron",
+    "url": "http://brandonaaron.net"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brandonaaron/jquery-spellcheck.git"
+  },
+  "homepage": "https://github.com/brandonaaron/jquery-expandable",
```
Comment (kennynaoh) [inline ajax/libs/jquery-spellcheck/package.json]: @honestree wrong homepage

**Comment by honestree** [comment]
@kennynaoh 
I updated the homepage block in the PR!
please help me review it!

File: ajax/libs/jquery-spellcheck/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "0.2",
+  "author": {
+    "name": "Brandon Aaron",
+    "url": "http://brandon.aaron.sh/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brandonaaron/jquery-spellcheck.git"
+  },
+  "homepage": "http://plugins.jquery.com/project/spellcheck",
```
Comment (kennynaoh) [inline ajax/libs/jquery-spellcheck/package.json]: Broken url

**Comment by honestree** [comment]
@kennynaoh
I deleted the homepage block, since the url is broken. 
please help me review it!

**Comment by kennynaoh** [comment]
@honestree Please combine those three commit to one commit.

**Comment by honestree** [comment]
@kennynaoh I combined the commit message into one! please help me review it!

File: ajax/libs/jquery-spellcheck/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "version": "0.2",
+  "author": {
+    "name": "Brandon Aaron",
+    "url": "http://brandon.aaron.sh/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brandonaaron/jquery-spellcheck.git"
+  },
+  "auto-update": {
```
Comment (kennynaoh) [inline ajax/libs/jquery-spellcheck/package.json]: wrong

**Comment by honestree** [comment]
@kennynaoh I tried to fix it! please help me review this!

File: ajax/libs/jquery-spellcheck/package.json (hunk 3 above)
Comment (kennynaoh) [inline ajax/libs/jquery-spellcheck/package.json]: @honestree No dash here

**Comment by honestree** [comment]
@kennynaoh I removed the dash!

**Comment by kennynaoh** [comment]
No, you don't remove it.

**Comment by honestree** [comment]
@kennynaoh I misunderstand you meaning, but now I've removed the dash for sure!

**Comment by kennynaoh** [comment]
LGTM, ping @cdnjs/intern2

File: ajax/libs/jquery-spellcheck/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "url": "git://github.com/brandonaaron/jquery-spellcheck.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brandonaaron/jquery-spellcheck.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery-spellcheck*"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-spellcheck/package.json]: Wrong rule. It'll fetch nothing.

Comment (honestree) [inline ajax/libs/jquery-spellcheck/package.json]: @pvnr0082t What should i do to  fix this?

Comment (pvnr0082t) [inline ajax/libs/jquery-spellcheck/package.json]: @honestree Try to adjust the rule. You can try to clone [git auto-update](https://github.com/cdnjs/autoupdate) at the same path of cdnjs in your computer, then run `npm install` to install dependencies you need. Run `./autoupdate [library_name]` and you'll know files that auto-updater will fetch.

**Comment by honestree** [comment]
@pvnr0082t  I tried to fix it! please help me review this!

File: ajax/libs/jquery-spellcheck/package.json (hunk 4 above)
Comment (honestree) [inline ajax/libs/jquery-spellcheck/package.json]: I used **git clone https://github.com/cdnjs/autoupdate.git**
and **cd autoupdate**
and then **npm install**
but it failed to install the dependencies.

**Comment by pvnr0082t** [comment]
LGTM
@kennynaoh You don't approve your part.

**Comment by PeterDaveHello** [comment]
You are NOT the author of this library, why did you check the box of `I'm the author of this library `?

**Pull request merged**